notes/1.0.0.markdown

```diff
@@ -1,5 +1,137 @@
 See [sbt 1.0 roadmap and beta-1][sbt-1-0-roadmap] for scheduled timeline.
 
+### sbt 1.0.0-M6
+
+This is the beta-2 release of sbt 1.0.
+
+### Changes since sbt 1.0.0-M5
+
+#### Features, fixes, changes with compatibility implications
+
+We are working with Scala Center to provide [an automatic migration tool][sbt-migration-rewrites].
```

Comment:
"The Scala Center is working with Lightbend to provide an automatic migration tool"?